lib/python/rose/tests/test_ancils/unicode.txt

```diff
@@ -0,0 +1,301 @@
+UTF-8 decoder capability and stress test
+----------------------------------------
+
+Markus Kuhn <http://www.cl.cam.ac.uk/~mgk25/> - 2015-08-28 - CC BY 4.0
```

Comment:
I think this is an acceptable licence for use to just bung this file in: https://creativecommons.org/licenses/by/4.0/deed.ast

Reply:
May need to add it to `ATTRIBUTIONS.md` to comply.

Reply:
Done